client/components/core/badge/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { privateApis } from '@wordpress/components';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import React from 'react';
+
+const CoreBadge = ( { children }: { children: React.ReactNode } ) => {
+	const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
+		'I acknowledge private features are not for use in themes or plugins and doing so will break in the next version of WordPress.',
+		'@wordpress/components'
```

Comment:
I guess that, since we need to pretend to be a core package, using `@wordpress/components` here would be ok?

@Automattic/team-calypso

Reply:
I'm OK with this here, but I don't understand why we're doing it in a wrapper component instead of using the private API directly.